libs/models/constants.py

```diff
@@ -0,0 +1,171 @@
+VALID_CURRENCIES = [
+    "AFN",
+    "ALL",
+    "DZD",
+    "USD",
+    "EUR",
+    "AOA",
+    "XCD",
+    "ARS",
+    "AMD",
+    "AWG",
+    "AUD",
+    "AZN",
+    "BSD",
+    "BHD",
+    "BDT",
+    "BBD",
+    "BYN",
+    "BZD",
+    "XOF",
+    "BMD",
+    "BTN",
+    "INR",
+    "BOB",
+    "BOV",
+    "BAM",
+    "BWP",
+    "NOK",
+    "BRL",
+    "BND",
+    "BGN",
+    "BIF",
+    "CVE",
+    "KHR",
+    "XAF",
+    "CAD",
+    "KYD",
+    "CLF",
+    "CLP",
+    "CNY",
+    "COP",
+    "COU",
+    "KMF",
+    "CDF",
+    "NZD",
+    "CRC",
+    "HRK",
+    "CUC",
+    "CUP",
+    "ANG",
+    "CZK",
+    "DKK",
+    "DJF",
+    "DOP",
+    "EGP",
+    "SVC",
+    "ERN",
+    "ETB",
+    "FKP",
+    "FJD",
+    "XPF",
+    "GMD",
+    "GEL",
+    "GHS",
+    "GIP",
+    "GTQ",
+    "GBP",
+    "GNF",
+    "GYD",
+    "HTG",
+    "HNL",
+    "HKD",
+    "HUF",
+    "ISK",
+    "IDR",
+    "XDR",
+    "IRR",
+    "IQD",
+    "ILS",
+    "JMD",
+    "JPY",
+    "JOD",
+    "KZT",
+    "KES",
+    "KPW",
+    "KRW",
+    "KWD",
+    "KGS",
+    "LAK",
+    "LBP",
+    "LSL",
+    "ZAR",
+    "LRD",
+    "LYD",
+    "CHF",
+    "MOP",
+    "MGA",
+    "MWK",
+    "MYR",
+    "MVR",
+    "MRU",
+    "MUR",
+    "XUA",
+    "MXN",
+    "MXV",
+    "MDL",
+    "MNT",
+    "MAD",
+    "MZN",
+    "MMK",
+    "NAD",
+    "NPR",
+    "NIO",
+    "NGN",
+    "OMR",
+    "PKR",
+    "PAB",
+    "PGK",
+    "PYG",
+    "PEN",
+    "PHP",
+    "PLN",
+    "QAR",
+    "MKD",
+    "RON",
+    "RUB",
+    "RWF",
+    "SHP",
+    "WST",
+    "STN",
+    "SAR",
+    "RSD",
+    "SCR",
+    "SLE",
+    "SGD",
+    "XSU",
+    "SBD",
+    "SOS",
+    "SSP",
+    "LKR",
+    "SDG",
+    "SRD",
+    "SZL",
+    "SEK",
+    "CHE",
+    "CHW",
+    "SYP",
+    "TWD",
+    "TJS",
+    "TZS",
+    "THB",
+    "TOP",
+    "TTD",
+    "TND",
+    "TRY",
+    "TMT",
+    "UGX",
+    "UAH",
+    "AED",
+    "USN",
+    "UYI",
+    "UYU",
+    "UZS",
+    "VUV",
+    "VEF",
+    "VED",
+    "VND",
+    "YER",
+    "ZMW",
+    "ZWL",
+]
```

Comment:
are these the currencies for the zones we have or all available currencies?

Reply:
All available currencies in the world.